app/views/dashboard/index.html.erb

```diff
@@ -1,13 +1,25 @@
 <%= javascript 'dashboard' %>
 <% title _('Overview') %>
 <%= title_actions dashboard_actions %>
+
 <div class="row">
-  <div id='dashboard' class="gridster col-md-12">
+  <div id='dashboard' class="row gridster col-md-12">
+  <% if User.current.widgets.empty? %>
+    <%= _("No widgets defined. Why not add the defaults from the menu?") %>
+  <% else %>
     <ul>
-      <%= widget_list.map do |widget|
-          content_tag(:li, render(widget.id), :data=>{:id => widget.id, :name => widget.name || widget.id, :row => widget.row, :col => widget.col, :sizex=>widget.sizex, :sizey => widget.sizey, :hide=>widget.hide})
-        end.join(' ').html_safe
-    %>
+      <% User.current.widgets.each do |widget| %>
+        <%= content_tag(:li, widget_data(widget)) do %>
```

Comment:
you should rebase your version, there was a recent update to the dashboard index.